src/Makefile.in

```diff
@@ -241,6 +241,10 @@ ICEBLOCKS_EXE		= $(BINDIR)/miceblocks$(EXESUFFIX)
 ICEBLOCKS_OBJ		= miceblocks$(OBJSUFFIX)
 ICEBLOCKS_SO		= $(LIBDIR)/libmiceblocks$(LIBSUFFIX)
 
+JAN4X4_EXE = $(BINDIR)/mjan$(EXESUFFIX) # A simple games
+JAN4X4_OBJ = mjan$(OBJSUFFIX)
+JAN4X4_SO  = $(LIBDIR)/libmjan$(LIBSUFFIX)
```

Comment:
Don't think you need the last line if you don't have other includes other than gamesman.h

Reply:
sg